app/controllers/api/v0/articles_controller.rb

```diff
@@ -70,6 +75,25 @@ def me
 
       private
 
+      INDEX_ATTRIBUTES_FOR_SERIALIZATION = %i[
```

Comment:
you'll notice this pattern in the entire PR

Reply:
`fast_jsonapi` is in the back of my mind for API v1, as are JSON Schema and OpenAPI with things like https://github.com/interagent/committee

Reply:
fast JSON has been something I have been dying to use since working with Elasticsearch requires so much serialization I think it could really speed things up when we hit higher volumes.